core/src/main/scala/com.snowplowanalytics.snowplow.event.recovery/config/SelfDescribingBadRow.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2018-2020 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and
+ * limitations there under.
+ */
+
+package com.snowplowanalytics.snowplow.event.recovery.config
+
+import com.snowplowanalytics.snowplow.badrows.BadRow
+
+/**
+  * Defines a standard wrapper for a self-describing bad row.
+  */
+case class SelfDescribingBadRow(schema: String, data: BadRow)
```

Comment:
Why not have all three case classes (`Conf`, `FlowConfig` and `SelfDescribingBadRow`) in one file?